### PR TITLE
Bumping some dependencies

### PR DESCRIPTION
**Pull request by mistercrunch**
Most notably Flask AppBuilder to 1.9.0

### CODE REVIEW

File: superset/assets/package.json [hunk 1]
```diff
@@ -93,28 +93,28 @@
     "supercluster": "https://github.com/georgeke/supercluster/tarball/ac3492737e7ce98e07af679623aad452373bbc40",
     "topojson": "^1.6.22",
     "urijs": "^1.18.10",
-    "victory": "^0.18.4",
+    "victory": "^0.21.0",
```
Comment (ascott) [inline superset/assets/package.json]: we could remove this altogether, we don't use it anywhere.

**Comment by ascott** [comment]
LGTM

File: superset/assets/package.json (hunk 1 above)
Comment (mistercrunch) [inline superset/assets/package.json]: Ok good to know. I'll merge this as is and will run `depcheck` to look for other unused deps and send another PR to clean that up.

Comment (ascott) [inline superset/assets/package.json]: 👍

**Pull request merged**